src/start.js

```diff
@@ -3,10 +3,32 @@ import 'express-async-errors'
 import logger from 'loglevel'
 import {getRoutes} from './routes'
 import config from 'config'
+const  swaggerJSDoc = require('swagger-jsdoc')
+const swaggerUi = require('swagger-ui-express')
+
 
 function startServer({port = process.env.PORT} = {}) {
   const app = express()
 
+  const swaggerOptions = {
+    swaggerDefinition: {
+    info: {
+        title: 'TFT API',
+        version: '1.0.0',
+        description: 'TeamFightTactics gameplay Information',
+        contact: {
+        name: 'Blake Archer',
+        email: 'blarcher31@gmail.com'
+        },
+        server: ['http://localhost:8080']
+    }
+    },
+    apis: ['start.js']
+}
+const swaggerDocs = swaggerJSDoc(swaggerOptions)

```

Comment:
Does this work? I'll have to take a look at the package you pulled in to see how to set this up.